packages/component-model-survey-response/entities/surveyResponse/index.js

```diff
@@ -0,0 +1,48 @@
+const BaseModel = require('@pubsweet/base-model')
+
+class SurveyResponse extends BaseModel {
+  static get tableName() {
+    return 'survey_response'
+  }
+
+  static get schema() {
```

Comment:
Should this be hard-coded?

Reply:
This is consistent with the other models, for simple models like this, the ORM is pretty much just this schema object wrapped with the `BaseModel`. Because it's the core of what the object is,  I don't think we should be abstracting the JSON out of this file.